testdata/ostrusted/Issue4.java

```diff
@@ -0,0 +1,7 @@
+interface Issue4Interface {
```

Comment:
There is no Issue number 4 corresponding to this test file. Either file an issue or use a different name.
The file should start with "//@skip-test" until the test is fixed.